modules/trustxBidAdapter.js

```diff
@@ -1,165 +1,147 @@
-const utils = require('src/utils.js');
-const bidfactory = require('src/bidfactory.js');
-const bidmanager = require('src/bidmanager.js');
-const adloader = require('src/adloader');
-const adaptermanager = require('src/adaptermanager');
-const CONSTANTS = require('src/constants.json');
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'trustx';
+const ENDPOINT_URL = '//sofia.trustx.org/hb';
+const TIME_TO_LIVE = 360;
+const ADAPTER_SYNC_URL = '//sofia.trustx.org/push_sync';
+const LOG_ERROR_MESS = {
+  noAuid: 'Bid from response has no auid parameter - ',
+  noAdm: 'Bid from response has no adm parameter - ',
+  noBid: 'Array of bid objects is empty',
+  noPlacementCode: 'Can\'t find in requested bids the bid with auid - ',
+  emptyUids: 'Uids should be not empty',
+  emptySeatbid: 'Seatbid array from response has empty item',
+  emptyResponse: 'Response is empty',
+  hasEmptySeatbidArray: 'Response has empty seatbid array',
+  hasNoArrayOfBids: 'Seatbid from response has no array of bid objects - '
+};
+const bidsRequestMap = {};
+export const spec = {
+  code: BIDDER_CODE,
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.uid && bid.adUnitCode);
```

Comment:
`bid.adUnitCode` is not required here to validate bid reqeust. It is part of prebid-core and already tested there.